bag/datasets/bag/queries.py

```diff
@@ -136,6 +136,11 @@ def _basis_openbare_ruimte_query(
 
     sort_fields = ['_score', 'naam.keyword']
 
+    _fields = ['naam', 'naam_nen', 'postcode']
+    from search.views import FEATURE_1
+    if features & FEATURE_1:
+        _fields.append('naam_no_ws')
```

Comment:
Alleen een vraag: 

_fields.append('naam_no_ws') => deze wordt toegevoegd bij feature 1. 

In de _fields list is een item met 'naam', die dezelfde value bevat als naam_no_ws. Klopt dit?

Reply:
Ja dat klopt. Ze gebruiken een verschillende analyzer, zowel bij indexering als query verwerking. 
naam gebruikt analyzers.adres 
naam_no_ws gebruikt analyzers.straat_no_ws

De laatste stript de whitespace weg, zodat  "Lange brug" matcht met "Langebrugsteeg" en "Janvoerman" matcht met "Jan Voermanstraat"